package.json

```diff
@@ -25,10 +25,10 @@
     "prepare": "yarn build",
     "start": "yarn storybook",
     "storybook": "start-storybook -p 3000 -s public/development",
-    "test": "jest --detectOpenHandles",
+    "test": "cross-env NODE_OPTIONS='--max-old-space-size=8096' jest --detectOpenHandles",
```

Comment:
I don't think `--detectOpenHandles` should be used outside of a manual flag when debugging, it has perf implications

Reply:
I removed the flag, and saw the following output
```
Test Suites: 49 passed, 49 total
Tests:       435 passed, 435 total
Snapshots:   241 passed, 241 total
Time:        72.575s
Ran all test suites.
Jest did not exit one second after the test run has completed.

This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
```
Although there is nothing detected.

Reply:
I've opened #732 to further investigate and track this issue. No need to hold up this PR